CHANGES.markdown

```diff
@@ -1,3 +1,7 @@
+# 2.1.5 (2016-08-03)
```

Comment:
You can leave this as `# master (unreleased)`. I'll rev the version for a new gem.